org.osgi.test.junit5/test.bndrun

```diff
@@ -48,9 +48,6 @@
 	junit-platform-engine;version='[1.6.2,1.6.3)',\
```

Comment:
Remove this file from your PR, It is being fixed in https://github.com/osgi/osgi-test/pull/172